.github/workflows/pr-validation.yml

```diff
@@ -147,6 +147,14 @@ jobs:
             tools:
               - 'tools/**'
 
+      - name: Set up QEMU
+        if: steps.filter.outputs.tools == 'true'
+        uses: docker/setup-qemu-action@v2
```

Comment:
Do you need QEMU? In main-build workflow (which builds docker images) we don't need it